src/SourceGenerators/Uno.UI.SourceGenerators/Content/Uno.UI.SourceGenerators.props

```diff
@@ -86,7 +86,9 @@
 
   <ItemGroup>
 
-	<ExcludeXamlNamespaces Include="win"/>
+	<ExcludeXamlNamespaces Include="win" />
+	<ExcludeXamlNamespaces Include="uwp" Condition="'$(UNO_UWP_BUILD)'!='true'" />
```

Comment:
Unsure how this can happend. The source generator can be invoked on a UWP build?